linux-create-vm.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+# Update and Upgrade apt
+
+sudo apt update
```

Comment:
We should probably wrap these in conditionals eventually too. If they are even needed?

Reply:
Why are you just running `sudo apt update` and `sudo apt upgrade`? What is the desired state you want and how would you test if you NEED to run these things? It's a hard one, but these and EVERYTHING should be in a conditional. Some are just harder to reason about sometimes.

Reply:
I don't think they are needed @stephenlauck, but I was going to work on this today when I get off work.

Reply:
My initial thought was to have update versions of the other packages we're installing, but I've read on it a little and what I'm seeing is that it's not recommended to do when you are writing a script to do one task. I don't think it's needed.